libs/hwes/eventstoredb/projections/custom/custom.go

```diff
@@ -0,0 +1,204 @@
+package custom
+
+import (
+	"context"
+	"fmt"
+	"github.com/EventStore/EventStore-Client-Go/esdb"
+	zlog "github.com/rs/zerolog/log"
+	"hwes"
+	"strings"
+	"telemetry"
+)
+
+// PersistentSubscriptionFailedCreationErrorCode https://github.com/EventStore/EventStore-Client-Go/blob/v1.0.2/esdb/types.go#L95
+const PersistentSubscriptionFailedCreationErrorCode = 6
+
+// EventStoreDBInternalEventPrefix https://developers.eventstore.com/server/v23.10/streams.html#reserved-names
+const EventStoreDBInternalEventPrefix = "$"
+
+type eventHandler func(ctx context.Context, evt hwes.Event) (error, esdb.Nack_Action)
+
+// CustomProjection can be used to develop own projections
+// A projection is an event sourcing pattern to build up
+// a read model based on the underlying event data.
+// IMPORTANT: The passed subscriptionGroupName identifies you subscription
```

Comment:
From what I could gather from my (admittedly brief) look at [the docs](https://developers.eventstore.com/server/v23.10/persistent-subscriptions.html#concepts), group names are not meant to identify subscriptions. In contrary, they are used to batch subscriptions together, and lets the server know which subscribers a message can be load-balanced between.